dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/maintenance/jdbc/JdbcMaintenanceStore.java

```diff
@@ -82,6 +82,23 @@ public int deleteZeroDataValues()
         return jdbcTemplate.update( sql );
     }
 
+    @Override
+    public int deleteInvalidRelationships()
+    {
+        String relationshipsSelect = "(select relationshipid from relationship where from_relationshipitemid is NULL " +
+            "OR to_relationshipitemid is NULL)";
+
+        /*
```

Comment:
Remove this inline javadoc, it is clear what is going on.

Reply:
👍